ci/ci/ci.py

```diff
@@ -85,7 +85,7 @@
 
     context = {
         'watched_branches': wb_configs,
-        'age': pretty_timestamp_age(datetime.datetime.now() - start_time),
```

Comment:
Is pretty_timestamp_age still used anywhere? If not, delete.

Reply:
yes. It's used by PR.